src/go/rpk/pkg/config/schema.go

```diff
@@ -48,19 +48,19 @@ type RedpandaConfig struct {
 	ID                         int                       `yaml:"node_id" json:"node_id"`
 	Rack                       string                    `yaml:"rack,omitempty" json:"rack"`
 	SeedServers                []SeedServer              `yaml:"seed_servers" json:"seed_servers"`
```

Comment:
Why not omitempty here?

Reply:
`node_id` can be 0.

`seed_servers: []` <- Should be present, if `"omitempty"` is set we would strip it.

See: https://redpandadata.slack.com/archives/C024H6DSU2H/p1658439198290169